### PR TITLE
Recycle Pose Pickles - Save Energy PR 

### PR DESCRIPTION
**Pull request by antoinefalisse**
@suhlrich this is finally ready I think. Below are the tests I've done. Seems to be working as expected.
Can you take another look? I had to change how we organize the docker files to be able to load the default settings in the containers. Also modified how we check for the metadata as compared to last time.

- [x] Test w/ OpenPose 
- [x] Test w/ HRNet
- [x] Verify that correct information is saved in main_settings.yaml for OpenPose
- [x] Verify that correct information is saved in main_settings.yaml for HRNet
- [x] Re-process OpenPose with same settings as saved +  verify what is saved in main_settings.yaml
- [x] Re-process OpenPose with different settings as saved +  verify what is saved in main_settings.yaml
- [x] Re-process HRNet with same settings as saved +  verify what is saved in main_settings.yaml
- [x] Re-process HRNet with different settings as saved +  verify what is saved in main_settings.yaml
- [x] Re-process old OpenPose session +  verify what is saved in main_settings.yaml
- [x] Re-process old hrnet session +  verify what is saved in main_settings.yaml

### CODE REVIEW

File: defaultOpenCapSettings.json
Comment (suhlrich) [inline defaultOpenCapSettings.json:1]: I'm ok leaving this as is, but do we want to be more specific about the parameter: openpose: {'resolution': "1x736"}}

**Review by suhlrich — COMMENTED** [review]
looks great. small comment

File: defaultOpenCapSettings.json
Comment (antoinefalisse) [reply defaultOpenCapSettings.json:1]: Will do as part of another PR with other cleaning items.

Comment (antoinefalisse) [reply defaultOpenCapSettings.json:1]: https://github.com/stanfordnmbl/opencap-core/issues/99

**Pull request merged**